.ebextensions/db.config

```diff
@@ -0,0 +1,3 @@
+option_settings:
+  aws:elasticbeanstalk:application:environment:
+    DB_ATLAS_CONNECTION: mongodb+srv://vvek87:koolpool@cluster0-nokp5.mongodb.net/mashbnb?retryWrites=true
```

Comment:
I'm pretty sure this is the issue you're facing, but if possible, you'll want to not track this, and possibly see about deleting this from your commit history